modules/bibauthorid/lib/bibauthorid.js

```diff
@@ -2418,19 +2429,32 @@ $(document).ready(function() {
             });
 
             $("form>#mergeButton").parent().on("submit", function(event) {
+
                 var bodyModel = ticketbox.app.bodyModel,
                     $form = $(this),
                     primaryCname = $form.find("input[name='primary_profile']").first().val(),
                     modalContent = Handlebars.templates.mergeConfirmation(),
                     modal =
-                    callbacks = null;
+                    callbacks = null;                 
 
                 callbacks = {
                     "show.bs.modal": function(e) {
                         var $confirmButton = $(e.currentTarget).find("a.confirm"),
                             $modal = $("#ticket-modal");
 
+                        if(gUserLevel == "guest"){
+                            // If user is not authenticated, ask for email
+                            $modal.find(".modal-body").append('You are about to submit a ticket. Please provide your e-mail address to confirm this action. We will contact you in case of questions.');
```

Comment:
Isn´t a bit more gentle "You are about to submit a ticket. Please, provide your e-mail address to confirm this action, so that we can contact you in case of questions."?